bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/UpnpDiscoveryService.java

```diff
@@ -102,6 +105,11 @@ protected void startScan() {
         }
         upnpService.getRegistry().addListener(this);
         upnpService.getControlPoint().search();
+
+        // Belkin WeMo devices can not be discovered by default search() function, but have to be searched by their
+        // named service.
+        ServiceType wemoBasicEvent = new ServiceType("Belkin", "basicevent");
+        upnpService.getControlPoint().search(new ServiceTypeHeader(wemoBasicEvent));
```

Comment:
Indeed, not the best solution, but I also don't see any sideeffects.
UPnP discovery will add all found devices to the registry, no matter if a specific binding is installed or not. These two lines just make sure that even the WeMo devices are discovered and add to the registry.
What I can try (and will do) is to call the UPnP search within the Binding initialisation.
But this would break the option to start a scan mannually within the UI.